src/reducer/helper/calendar.ts

```diff
@@ -0,0 +1,64 @@
+//#region > Imports
+
+//> Moment
+// A lightweight JavaScript date library for parsing,
+// validating, manipulating, and formatting dates.
+import moment from "moment";
+
+
+//> Models
+// Contains all models of the database.
+import * as models from "../database/models";
+
+//> Interfaces
+// Contains a calendar interface
+import { ICalendar } from "../database/helper/calendar";
+//#endregion
+
+//#region > Functions
+/**
+ * Get a merged calendar object.
+ *
+ * @returns {object} A calendar object containing the current calendar structure object and the calendar structure objects of all years.
```

Comment:
We allow up to 119 characters as this is the width of GitHub code review, anything longer requires horizontal scrolling which makes review more difficult. Documentation, comments, and docstrings should be wrapped at 79 characters.